### DISCUSSION TITLE
running multiple bots

### DISCUSSION THREAD

**Discussion by sSimuSs** [go-telegram-bot-api/telegram-bot-api#462] [Q&A]
Hello, is it possible to run multiple bots? Thank you) (with the same domain and port)

**Comment by Syfaro** [comment]
Yes, by specifying different paths on the same domain and port you can run an arbitrary number of bots. 

```text
https://your-bot.example:8443/bot1
https://your-bot.example:8443/bot2
```

Replacing the domain and port as needed (and probably setting the path to something harder to guess). Then each bot can listen on the correct path.

```go
updates1 := bot.ListenForWebhook("/bot1")
updates2 := bot.ListenForWebhook("/bot2")
```

**Reply by JoxonXaker** [reply]
If possible, give more detailed information, for example
